data-access/UserRepository.cs

```diff
@@ -184,15 +184,15 @@ public async Task<UserEditionPermissions> GetUserEditionPermissionsAsync(UserInf
 						UserPermissionQuery.GetQuery,
 						new
 						{
-							EditionId = user.editionId,
-							UserId = user.userId
+							editionUser.EditionId,
+							UserId = editionUser.userId ?? 1 // if userID is null we get the permissions for userId 1
```

Comment:
Why? Why do we get the permissions for user 1? This needs a better comment.

Reply:
I will add a better comment, thanks. Basically userId 1 holds the permissions that apply to an anonymous user (who would naturally have no user id).